nerdtree_plugin/fs_menu.vim

```diff
@@ -203,7 +203,7 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        let metadata = split(system('ls -ldh ' . shellescape(treenode.path.str())), '\n')
```

Comment:
@PhilRunninger, the sed option doesn't work very well with the proposed regex

```
ls -lh
805K 17 Feb 10:14 2017-02-14-Windsor-T21986.jpeg
```
with sed and the proposed regex the file name get corrupted
```
ls -l | sed -e :a -e 's/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/;ta'
823,989 17 Feb 10:14 2,017-02-14-Windsor-T21,986.jpeg
```